File: Java/Hardware.java
public class Hardware extends Product {

    /* Private Attribute */
    private String brand;
    private String model;

    /* Constuctor */
    public Hardware(String brand, String model) {
        this.brand = brand;
        this.model = model;
    }

    public Hardware() {

    }

    /* Setter dan Getter Price */
    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getBrand() {
        return brand;
    }

    /* Setter dan Getter Id Product */
    public void setModel(String model) {
        this.model = model;
    }

    public String getModel() {
        return model;
    }

}

File: Python/Hardware.py
from Product import Product

class Hardware(Product):

    #private attributes
    __brand = "Brand"
    __model = "Model"

    #Constructor
    def __init__(self, brand = "Brand", model = "Model"):
        self.__brand = brand
        self.__model = model
    
    #Setter dan Getter Brand
    def setBrand(self, brand):
        self.__brand = brand

    def getBrand(self):
        return self.__brand

    #Setter dan Getter Model
    def setModel(self, model):
        self.__model = model

    def getModel(self):
        return self.__model